levels/pages/faq/faq.jade

```diff
@@ -335,3 +337,48 @@ block content
                 <div class='block__elem3'></div>
             </div>
             ```
+
+        a(id="global-css-resets")
+        .raw-text
+          :markdown
+            ## I've heard that BEM does not recommend global CSS resets. Why?
+
+            > CSS resets is a practise making a good showing. Many frameworks first align anything and then
+            > apply their special styles. BEM does not recommend common resets. Why? And what we are supposed
+            > to do instead?
+
+            Nothing bad would happen to your blocks if you use common reset (well, except of some special cases below).
+            So, BEM does not prohibit to use them. But using them BEM-way would be more effective.
+
+            Common CSS reset is a set of CSS to be applied to document nodes and ensure that their default view is the
+            same in different browsers. In most cases the CSS rules are written for tag selectors and this is not recommended
+            in BEM (you can find a lot of explanation above).
+
+            Another point is that in BEM a block encapsulates everything which is needed for it to be displayed and function.
+            And this is why we call the BEM blocks independent. If the block does not look properly without a third-party
+            CSS being added onto the page, it cannot be called "independent§" that much.
+
+            Assuming this all, BEM recommends every block to reset itself. If you have `menu` block and `list` block both
```

Comment:
@varya please do. I talked in our team about resets and we don't use them, because styles that needs to be "normalized" just written in block without `@include` statement. Turns out, that there are not much styles, which needs to be normalized in one block/element.

You can add tip: `You can use style preprocessor to remove duplicate code...` (but still this will confront your statement, that blocks should be independent - now they depends on reset includes).